contracts/poap/src/execute.rs

```diff
@@ -392,6 +392,18 @@ where
             return Err(ContractError::MintDisabled {});
         }
 
+        // Check if this user have already minted a poap.
```

Comment:
By putting this check here, the admin and the minter are still able to mint more than a poap to each user. Is this behavior a desired one?

Reply:
Since I did not have any specifics on this behavior, I decided to give them the option of minting as many poaps as they want.  In any case, minting can be limited for them as well.
Keep in mind that even with the `MintTo` action it is possible to mint more than one POAP for a user, in this case, however, I would like to avoid checking if the user already has a POAP to save gas, since this is an operation performed by a trusted party.

Reply:
@manu0466 I believe we can consistently check that each user has only one POAP minted. As this is a **read** operation, it won't cost gas (gas is only used for **write** operations). This ensures that the behavior remains consistent.